google/cloud/spanner_v1/client.py

```diff
@@ -292,6 +292,36 @@ def __init__(
         self._nth_client_id = Client.NTH_CLIENT.increment()
         self._nth_request = AtomicCounter(0)
 
+        self._log_spanner_options()
+
+    def _log_spanner_options(self):
+        """Logs Spanner client options."""
+        host = "spanner.googleapis.com"
+        if self._emulator_host:
+            host = self._emulator_host
+        elif self._experimental_host:
+            host = self._experimental_host
+        elif self._client_options and self._client_options.api_endpoint:
+            host = self._client_options.api_endpoint
```

Comment:
Emulator host must be getting set somewhere else in the client object, Can't we  reuse it ?